types/three/examples/jsm/modifiers/EdgeSplitModifier.d.ts

```diff
@@ -0,0 +1,19 @@
+import { BufferGeometry } from '../../../src/Three';
+
+export class EdgeSplitModifier {
+    constructor();
+
+    /**
+     * @param geometry					The geometry to modify by splitting edges.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/three/examples/jsm/modifiers/EdgeSplitModifier.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.